src/pages/index.js

```diff
@@ -52,17 +54,21 @@ const Index = ({ data }) => {
               })}
             </div>
             <section className="pb-8">
-              <p>
+              <div className="grid grid-cols-2 md:grid-cols-4 justify-items-center md:justify-items-start gap-4 md:gap-0 text-center md:text-left">
+                <Link to={`${pathPrefix}/all-rules?size=10`}>
+                  <FontAwesomeIcon icon={faStar} /> All rules
+                </Link>
                 <a href={`${pathPrefix}/archived`}>
-                  <FontAwesomeIcon icon={faArchive} /> Show archived rules
+                  <FontAwesomeIcon icon={faArchive} /> Archived rules
                 </a>
                 <a
                   href="https://www.ssw.com.au/ssw/Standards/Default.aspx"
-                  className="old-rules-link"
+                  className="col-span-2 md:col-span-1"
```

Comment:
Delete the *old-rules-link* class from style.css if its not being used anymore

Reply:
Done!